### PR TITLE
[#170593603]Script for Food Order System Portal Added

### PR DESCRIPTION
**Pull request by anuragganjiwale**

### CODE REVIEW

**Comment by the-spectator** [comment]
@anuragganjiwale Please fix the indentation as per ruby style guideline.

File: food_order_system_portal.rb [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+  end
+
+   
+  
+  def get_cost
+
+    cost = @price.to_i * @quantity.to_i
+
+    return cost
```
Comment (the-spectator) [inline food_order_system_portal.rb]: remove explicit returns on the last line of method.

File: food_order_system_portal.rb [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+    selected_restaurant = show_restaurants()
+
+    index = selected_restaurant.to_i - 1
+
+    @restaurant_name = $restaurants_list[index]
+
+
+    dishname = get_dishname()
+
```
Comment (the-spectator) [inline food_order_system_portal.rb]: Use proper spacing

File: food_order_system_portal.rb [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+	def initialize()
+		
+		@customer_name = ""
+		@restaurant_name = ""
+		@amount = 0
+    @dishname = ""
+    @order_status = "Not Delivered"
+    @quantity = 0
+    @@order_identifier = @@order_identifier + 1
+
```
Comment (the-spectator) [inline food_order_system_portal.rb]: FIx indentation

File: food_order_system_portal.rb [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+
+  def check_dish_availability(dish_number)
+
+    check_array = $dish_available[@restaurant_name]
+    check = check_array[dish_number.to_i - 1]
+    check = check.values
+    check = check[0]
+
```
Comment (the-spectator) [inline food_order_system_portal.rb]: Can we use hash here?

File: food_order_system_portal.rb [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+    if check == "no"
+      return 1
+    else 
+      return 0
+    end
```
Comment (the-spectator) [inline food_order_system_portal.rb]: Do we really need to return 1 or 0. Can we use boolean values

File: food_order_system_portal.rb [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+    count = 0
+    puts "Available Restaurants are : "
+    for i in 0..1 do
+
+      count = count + 1
+      puts "#{count}. #{$restaurants_list[i]} "
+    end
+    puts "Select one restaurant : "
+
```
Comment (the-spectator) [inline food_order_system_portal.rb]: Use iterators like each or map to iterats

File: food_order_system_portal.rb (hunk 1 above)
Comment (anuragganjiwale) [reply food_order_system_portal.rb]: done

File: food_order_system_portal.rb [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+      puts
+      puts "Sorry Selected Dish is Not available !!!"
+      puts "Try Again"
+      dishname = get_dishname()
+    end
+
+  end
+
+
+
```
Comment (the-spectator) [inline food_order_system_portal.rb]: Only One line white space should be here. Applicate to other places also

**Pull request merged**